frontend/src/pages/offers/PalletsEditor.tsx

```diff
@@ -0,0 +1,131 @@
+import cx from 'classnames'
+import { FunctionComponent, useState } from 'react'
+import {
+  OfferQuery,
+  PalletDocument,
+  useCreateLineItemMutation,
+  usePalletLazyQuery,
+} from '../../types/api-types'
+import LineItemForm from './LineItemForm'
+interface Props {
+  pallets: OfferQuery['offer']['pallets']
+  initiateDeletePallet: (palletId: number) => void
+}
+
+/**
+ * This is a 2-column UI with a list of pallets on the left and details aobut
+ * each line-item on the right.
+ */
+const PalletsEditor: FunctionComponent<Props> = ({ pallets }) => {
+  // TODO move this to the URL?
```

Comment:
thought (non-blocking): yep, that would be good so people can share a link to a specific pallet

Reply:
Yeah exactly. In a future PR!